src/routes/safe/components/Apps/components/ManageApps.tsx

```diff
@@ -4,16 +4,18 @@ import React, { useState } from 'react'
 import appsIconSvg from 'src/routes/safe/components/Transactions/TxsTable/TxType/assets/appsIcon.svg'
 import AddAppForm from '../AddAppForm'
 import { SafeApp } from '../types'
+import { onAppAddedHandler, onAppToggleHandler, onAppRemovedHandler } from '../hooks/useAppList'
 
 const FORM_ID = 'add-apps-form'
 
 type Props = {
   appList: Array<SafeApp>
-  onAppAdded: (app: SafeApp) => void
```

Comment:
That's not how it was designed to work. We shouldn't use return types from a hook as a component prop type. We have already described what type of a function we need for it to work, it doesn't necessarily have to be the exact type we return from a hook

Reply:
changed back!